src/SlackMessages.ts

```diff
@@ -82,8 +80,20 @@ export function atEveryone() {
     return "<!everyone>";
 }
 
+function hasItems(arr: any[]) {
+    return arr != null && arr.length > 0;
+}
+
 /** Renders JSON representation of slack message. */
 export function render(message: SlackMessage, pretty: boolean = false): string {
+    if (hasItems(message.attachments)) {
```

Comment:
Is this necessary? Won't `for of` just not evaluate the body is the list is empty or null?